source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -1,25 +1,40 @@
 import select from 'select-dom';
+import oneEvent from 'one-event';
 
 import features from '.';
 
-function init(): void {
+function addSourceTypeToLink(link: HTMLAnchorElement): void {
+	const search = new URLSearchParams(link.search);
+	search.set('type', 'source');
+	link.search = String(search);
+}
+
+// If the dropdown exists, await it; If not, it probably already loaded
+async function onProfileDropdownLoad(): Promise<void> {
+	const dropdown = select('.Header details-menu[src^="/users/"] include-fragment');
+	if (dropdown) {
+		await oneEvent(dropdown, 'load');
```

Comment:
It doesn't seem to be enough for me. Maybe `load` is triggered a little before the content is added

```
refined-github.js:2891 Uncaught TypeError: Cannot read property 'before' of undefined
    at HTMLDetailsElement.handleMenuOpening (refined-github.js:2891)
```

<img width="810" alt="Screen Shot 2020-12-14 at 20 41 01" src="https://user-images.githubusercontent.com/1402241/102156986-afa20680-3e4c-11eb-8e66-8b0cdd7f048d.png">

Reply:
Should we use a `MutationObserver`?

Reply:
You can try `one-mutation` with a filter if there’s no other way. 

Does `await Promise.resolve()` fix it? It helps using the debugger with dom breakpoints to see when the elements are appended

Reply:
`one-mutation` works fine, we don't even need a filter or to watch the subtree since all the content is loaded at once. Tested on Firefox & Chromium.